src/scenes/Projects/Budget/ProjectBudget.tsx

```diff
@@ -0,0 +1,176 @@
+import { Breadcrumbs, makeStyles, Typography } from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { GQLOperations } from '../../../api';
+import { Breadcrumb } from '../../../components/Breadcrumb';
+import { ContentContainer as Content } from '../../../components/ContentContainer';
+import { useCurrencyFormatter } from '../../../components/Formatters/useCurrencyFormatter';
+import { Table } from '../../../components/Table';
+import {
+  useProjectBudgetQuery,
+  useUpdateProjectBudgetRecordMutation,
+} from './ProjectBudget.generated';
+
+const useStyles = makeStyles(({ spacing }) => ({
+  header: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'flex-end',
+  },
+  total: {
+    fontSize: '1.5rem',
+  },
+  tableWrapper: {
+    marginTop: spacing(4),
+  },
+  toolbar: {
+    padding: spacing(2),
+    paddingBottom: spacing(1),
+  },
+}));
+
+export const ProjectBudget = () => {
+  const { projectId } = useParams();
+  const classes = useStyles();
+  const formatCurrency = useCurrencyFormatter();
+
+  const { data, loading, error } = useProjectBudgetQuery({
+    variables: { id: projectId },
+  });
+  const [updateBudgetRecord] = useUpdateProjectBudgetRecordMutation();
+
+  const projectName = data?.project.name.value;
+  const canReadBudget = data?.project.budget.canRead;
+  const canEditBudget = data?.project.budget.canEdit;
+  const budget = data?.project.budget.value;
+  const budgetRecords = budget?.records ?? [];
+
+  interface BudgetRowData {
+    id: string;
+    organization: string;
+    fiscalYear: string;
+    amount: string;
+    canEdit: boolean;
+  }
+
+  const rowData = budgetRecords.reduce((rows: BudgetRowData[], record) => {
+    const { amount, fiscalYear, id, organization } = record;
+    const { value: dollarAmount, canEdit } = amount;
+    const { value: year } = fiscalYear;
+    const orgName = organization.value?.name.value;
+
+    const row = {
+      id,
+      organization: orgName ?? '',
+      fiscalYear: String(year),
+      amount: String(dollarAmount),
+      canEdit,
+    };
+    return rows.concat(row);
+  }, []);
+
+  const columns = [
+    {
+      title: 'ID',
+      field: 'id',
+      hidden: true,
+    },
+    {
+      title: 'Organization',
+      field: 'organization',
+      editable: 'never' as const,
```

Comment:
FYI you could do `as const` on the root columns list definition instead of each time here. It applies to all inner declarations.